core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -476,6 +573,32 @@ public Sender build() {
             return sender;
         }
 
+        /**
+         * Disables automatic flushing of buffered data.
+         * <p>
+         * Sender buffers data locally before sending it to a server. This method disables automatic flushing and
+         * you must explicitly call {@link #flush()} to send buffered data to a server. It also disables auto-flushing
+         * on close. This gives you a fine control over batching behaviour.
```

Comment:
I didn't get to that part yet, but if the Java client now supports transactional requests, it's probably worth a mention here in the API docs about that feature.